docs/runbooks/SSPCommonTemplatesModificationReverted.md

```diff
@@ -0,0 +1,19 @@
+# SSPCommonTemplatesModificationReverted
+
+## Meaning
+
+This alert indicates that one or more common templates were changed by the ssp operator as part of its reconciliation procedure.
+
+## Impact
+
+If any changes were made to a common template manually or by a script they were reverted by the operator.
```

Comment:
I think this part should be a part of "Meaning" section since it explains why this alert was fired. In "Impact" session, we should state which kind of consequences this problem can lead.

Reply:
What about something like "You lose the change you made manually on the templates"

Reply:
Ok,  is the new version ok?